torch/_dynamo/variables/builtin.py

```diff
@@ -1425,6 +1430,12 @@ def _unimplemented():
         if isinstance(left, TensorVariable):
             from .builder import wrap_fx_proxy_cls
 
+            if op is operator.is_:
+                return ConstantVariable.create(
+                    id(left.as_proxy().node.meta["example_value"])
+                    == id(right.as_proxy().node.meta["example_value"])
```

Comment:
I'm OK with having the invariant that the identity of example_value must match the identity of the real tensor exactly, but I'm not 100% sure we always get it right